src/viur/core/modules/script.py

```diff
@@ -63,12 +59,16 @@ class ScriptLeafSkel(BaseScriptAbstractSkel):
 
     name = StringBone(
         descr="Filename",
-        vfunc=lambda value: not FILE_PATTERN.match(value),
+        required=True,
+        vfunc=lambda value:
+            None if File.is_valid_filename(value) and value.endswith(".py") and value.removesuffix(".py")
+            else "Filename is invalid or doesn't have a '.py'-suffix",
     )
 
     script = RawBone(
         descr="Code",
-        indexed=False
+        indexed=False,
+        required=True,
```

Comment:
On EVERY filesystem, even on windows, I can create an empty file. Why should this no longer by possible in ViUR.

```suggestion
```